ai_ref_kits/visual_gen_studio/streamlit_app.py

```diff
@@ -0,0 +1,25 @@
+import streamlit as st
+from streamlit_helper import (
+    init_session, check_fastapi_running, render_landing_page, render_scene_generation_page, render_image_generation_page,
+    apply_custom_styling 
+)
+
+# ------------------ Page Config ------------------

```

Comment:
Is it what is usually done with streamlit? No functions?

Reply:
Are you suggesting that I encapsulate the logic in streamlit_app.py into a main() function for better structure and readability, even though most of the functionality is modularized in streamlit_helper.py?

Reply:
I'm just thinking out loud. What is the usual approach here?